extensions/amazon-lambda-http/runtime/pom.xml

```
@@ -13,7 +13,7 @@
 
     <artifactId>quarkus-amazon-lambda-http</artifactId>
     <name>Quarkus - Amazon Lambda HTTP - Runtime</name>
-    <description>Allows Java applications written for a servlet container to run in AWS Lambda</description>
+    <description>Allows applications written for a servlet container to run in AWS Lambda</description>
```

```suggestion
    <description>Allow applications written for a servlet container to run in AWS Lambda</description>
```
You made the rules, man!